src/java.desktop/share/classes/javax/swing/plaf/synth/SynthGraphicsUtils.java

```diff
@@ -665,9 +670,17 @@ static void paintText(Graphics g, SynthMenuItemLayoutHelper lh,
                 g.setColor(lh.getStyle().getColor(
                         lh.getContext(), ColorType.TEXT_FOREGROUND));
                 g.setFont(lh.getStyle().getFont(lh.getContext()));
+
+                int mnemIndex = lh.getMenuItem().getDisplayedMnemonicIndex();
+                // Check to see if the Mnemonic should be rendered in GTK.
+                if ((UIManager.getLookAndFeel().getName().contains("GTK"))
+                    && SynthLookAndFeel.isMnemonicHidden() == true) {
```

Comment:
I was not expecting this check for GTK to still be here.

Reply:
I missed this to update last time.
Updated now to check using alt press property.